pandas/tseries/frequencies.py

```diff
@@ -57,15 +57,15 @@
 
 _offset_to_period_map = {
     "WEEKDAY": "D",
-    "EOM": "M",
-    "BM": "M",
+    "EOM": "ME",
+    "BM": "ME",
     "BQS": "Q",
     "QS": "Q",
     "BQ": "Q",
     "BA": "A",
     "AS": "A",
     "BAS": "A",
-    "MS": "M",
+    "MS": "ME",
```

Comment:
seeing as the values are period aliases, shouldn't they stay as they are? e.g. here `"MS": "M"`

Reply:
Thanks, I completely agree. I will keep the `"MS": "M"` which was here originally.